src/util/utils.js

```diff
@@ -628,3 +630,31 @@ export const getYesterdayDateRoundDownHour = () => {
   const yesterday = date.setDate(date.getDate() - 1)
   return new Date(yesterday).toISOString()
 }
+
+export const getMonthName = (month: number, short: boolean = false) => {
+  return short ? MONTH_NAMES_SHORT[month] : MONTH_NAMES[month]
+}
+
+export const formatExpDate = (expiration: Date, short: boolean = false) => {
+  const expirationDate = new Date(expiration)
+  return `${getMonthName(expirationDate.getMonth())}  ${expirationDate.getDate()}, ${expirationDate.getFullYear()}`
+}
+
+export const networkAlert = (): boolean => {
+  return Alert.alert(s.strings.network_alert_title, s.strings.network_alert_text)
+}
+
+export const getFeeDisplayed = (number: number) => {
```

Comment:
Why not just always use `toFixed(2)`?

Reply:
we need to show 2 numbers after the dot or all as needed